acsylla/_cython/session/session.pyx

```diff
@@ -47,6 +47,75 @@ cdef class Session:
         finally:
             cass_future_free(cass_future)
 
+    cdef const CassSchemaMeta* schema_meta(self):
+        if self._schema_meta:
+            cass_schema_meta_free(self._schema_meta)
+        self._schema_meta = cass_session_get_schema_meta(self.cass_session)
+        return self._schema_meta
+
+    def version(self):
+        cdef CassVersion v = cass_schema_meta_version(self.schema_meta())
+        return v.major_version, v.minor_version, v.patch_version
+
+    def snapshot_version(self):
+        return cass_schema_meta_snapshot_version(self.schema_meta())
+
+    def get_client_id(self):
```

Comment:
we would need to provide some coverage for the new functions, at least for checking the happy path.

Reply:
Yes, sure